apps/poker-roll/src/app/homePage.jsx

```diff
@@ -1,29 +1,41 @@
 import * as React from 'react';
 import BottomNav from "./bottomNav";
+import SessionForm from './startSession';
 import { 
     BrowserRouter as Router,
     Outlet,
     Switch,
     useLocation
 } from "react-router-dom";
 
+// import { ThemeProvider, createTheme } from '@mui/material/styles';
+
+// const darkTheme = createTheme({
+//     palette: {
+//       mode: 'dark',
+//     },
+//   });
+
 
 const Home = (props) => {
     
 
     return (
-        <div>
-            <div className="header">
-                <h1>Poker Roll</h1>
-            </div>
-            <div>
-                <Outlet />
-            </div>
             <div>
-                <BottomNav />
+                <div>
+                    <h1>Poker Roll</h1>
+                </div>
+                <div>
+                    <SessionForm />
```

Comment:
Don't worry about it right now but probably want to make this it's own `newSession` route or something at some point. Notice right now when you navigate to the other pages this form still shows up since this component wraps all pages.